### ISSUE TITLE
Recommended lab durations

### ISSUE THREAD

**Issue by vitaly-livshits** [MicrosoftLearning/AZ-203-DevelopingSolutionsforMicrosoftAzure#64]
I would like to ask other MCTs to share your experience with the duration of labs. 

My personal experience has been that suggested durations are generally too tight. There may thus be value in adjusting them upwards to make them more realistic. 

What do you think?

**Comment by adnkai** [comment]
I'm yet to teach the course but from what I've seen so far it was ok when I did the labs myself. I'll give more feedback next week when I have the student doing them.

**Comment by seesharprun** [comment]
I'm going to wait on more MCT feedback before recommending an action here.

Once we have a few more MCTs, I will put together a table of lab times and open that up for discussion.

**Comment by adnkai** [comment]
Sorry it took so long.
I held my course now and did some of the labs. The lab duration as it is recommended works fine for me when I do the labs. 
However, having 8 students doing the labs blows up the time requirements. There _will_ be issues for students which causes the overall time to rise significantly.

I don't think the lab requirements should account for such situations though.

**Comment by Usamawahabkhan** [comment]
sorry it took way too long to complete such lengthy labs.

**Comment by Usamawahabkhan** [comment]
we must need to create MCT demo repo as well. Please help on this node

**Comment by seesharprun** [comment]
I would recommend opening a separate issue for any discussion on demos. Today, demos are not hosted on GitHub for this course.

As far as lab length is concerned, I'm seeing clear feedback to trim some of the existing labs down and ensure any future labs are shorter in length and less complex.

When designing labs, we always try to find a balance between complexity, length, and covering as many objectives as possible.
